Doc/c-api/sys.rst

```diff
@@ -347,12 +347,14 @@ accessible to C code.  They all work with the current interpreter thread's
    :class:`Exception` (other errors will not be silenced).
 
    .. c:namespace:: NULL
-   .. c:type:: int (*Py_AuditHookFunction) (const char *, PyObject *, void *)
-
-   The hook function is of type :c:expr:`int (*)(const char *event, PyObject
-   *args, void *userData)`, where *args* is guaranteed to be a
-   :c:type:`PyTupleObject`. The hook function is always called with the GIL
-   held by the Python interpreter that raised the event.
+   .. c:type:: int (*Py_AuditHookFunction) (const char *event, PyObject *args, void *userData)
+
+      The type of the hook function.
+      *event* is (the event argument passed to PySys_Audit() or PySys_AuditTuple()),
```

Comment:
Wait, I used parentheses to denote that it is a preliminary text, and that you should rewrite it in good English. Perhaps it is even better to describe parameters not in terms of PySys_Audit() or PySys_AuditTuple(), but in more general terms. Look at `sys.audit()`. Perhaps "*event* is a C string identifying the event" is enough.

Reply:
Ah, I recognized "(...)" as a placeholder, not the more elaborate parenthesized phrasing. Will get back to it.

Reply:
> Perhaps "_event_ is a C string identifying the event" is enough.

I thought about that, but am not familiar enough with the C API documentation to know that descriptions of function prototypes don't simply rely on the actual prototype.